src/PeopleSDKAdapter.js

```diff
@@ -54,15 +54,21 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
   }
 
   /**
-   * Returns an observable that emits person data of the access token bearer.
+   * Returns an observable that emits only once of person data of the access token bearer.
```

Comment:
I would prefer to call out that the observable completes in a separate line.
```suggestion
   * Returns an observable that emits person data of the access token bearer.
   * The observable will emit once and then complete.
```